app/routes/users.js

```diff
@@ -27,7 +27,7 @@ module.exports = di => {
     );
     router.put(
         '/:id',
-        [uuidValidator('id'), validate],
+        [uuidValidator('id'), userValidator.update, validate],
```

Comment:
I don't understand why we have all these things in TODO app?

Reply:
Are we using it on frontend?

Reply:
there is no front-end